src/gitlab/functions/e2e-start-pipeline-button.ts

```diff
@@ -1,32 +1,48 @@
 import { isFrontend } from './is-frontend';
+import { isApp } from './is-app';
 import { GitlabPipelines } from '../services/gitlab-api/pipelines-api';
 import { GitlabJobs } from '../services/gitlab-api/jobs-api';
 import { createHashString } from './hash-variables';
 
+const FrontendImageJob = {
+    name: 'exponea docker image',
+    imageRegex: /digest: (sha256:[^ ]*) size: \d+/gm,
+    imageUrlParam: 'fe_version'
+}
+
+const AppImageJob = {
+    name: 'prod image',
+    imageRegex : /^Built Docker image\:.*digest:.*(sha256\:.+)$/m,
+    imageUrlParam: 'be_version'
+};
+
 export function createRunE2eButton(mergeRequestId: number) {
-    if (!isFrontend()) {
+    if (!isFrontend() && !isApp()) {
         return Promise.resolve();
     }
 
     if (mergeRequestId == null) {
         return Promise.resolve();
     }
 
+    const imageJob = isFrontend() ? FrontendImageJob : AppImageJob;
+
     return GitlabPipelines.getPipelinesForMR(mergeRequestId).then(pipelines => {
         if (!pipelines || pipelines.length === 0) {
         } else {
             const pipeline = pipelines[0];
             GitlabJobs.getJobsForPipeline(pipeline.id).then(jobs => {
                 if (jobs) {
-                    const job = jobs.filter(job => job.name === 'exponea docker image')[0];
-
+                    const job = jobs.filter(job => job.name === imageJob.name)[0];
```

Comment:
NIT: Use `Array.find` instead of `Array.filter`.

Reply:
sure, updated